src/core/filters/filterregistry.js

```diff
@@ -73,19 +72,14 @@ export default class FilterRegistry {
   }
 
   /**
-   * Gets the facet filter string to send in a search query.
-   * @returns {string}
+   * Gets the facet filters as an array of SimpleFilters to send to the answers-core.
+   *
+   * @returns {SimpleFilter[]}
```

Comment:
This is an answers-core `SimpleFilter`, correct? If so, can we indicate that the model is from the core and not from the SDK? In my adapter PRs, I put comments such as 'from answers-core' to indicate it